website/docs/Examples/EMGProjects/EMG_Joystick.md

```diff
@@ -0,0 +1,67 @@
+
+| Channel | Direction              |
+| ------- | ---------------------- |
+| 1       | - X (left)      |
+| 2       | + X (right)     |
+| 3       | + Y (up)        |
+| 4       | - Y (down)      |
+
+**Important**: The most complicated part of this tutorial can be finding the right placement for each set of electrodes. It is important to find the muscle locations that are the easiest for the user to precisely control. You may need to play around with different locations before finding your optimal electrode setup.
+
+## Step 2: GUI Setup
+
+Start by streaming data using the GUI. Follow this [getting started tutorial](../../Software/OpenBCISoftware/01-OpenBCI_GUI.md/#using-the-openbci-gui) to get the board streaming. You will need to open the EMG Joystick and EMG Settings widgets. 
+
+### EMG Settings
+
+This widget contains the tuneable parameters used for the threshold algorithm that classifies how active each channel is. These parameters should be adjusted to get optimal classification accuracy. The output value is a normalzed value mapped between the lower threshold and upper threshold (between 0-1)
+
+![EMG Settings Screenshot](../../assets/TutorialImages/EMG_settings.png)
+
+| Parameter | Definition              |
+| ------- | ---------------------- |
+| Smooth       | Controls how many seconds of data to average over. This reduces the effect of outliers in the dataset.    |
+| uV Limit     | Upper limit for the EMG signal.     |
+| Creep+       | Adjustment speed for upper threshold. How fast it should decrease if not triggered.     |
+| Creep-       | Adjustment speed for lower threshold. How fast it should increase if not triggered.      |
+| Min ΔuV      | Minimum difference awllowed between upper and lower thresholds.       |
+| Low Limit    | Minimum lower threshold allowed.      |
+
+A futher description of these parameters can be found on the [GUI Widgets options](../../Software/OpenBCISoftware/02_GUI_Widget_Guide.md/#options) page.
+
+### EMG Joystick Widget
+
+This widget has a marker that moves in the direction of the activated EMG channels. It is a weighted average of the EMG channel outputs. This visualization should be used in combination with the EMG settings to find your ideal EMG joystick setup.
+
+![EMG Joystick Screenshot](../../assets/TutorialImages/EMG_joystick.png)
+
+## Step 3: Stream Data Using Networking Widget
+
+Follow the [Networking Tutorial](../../Software/OpenBCISoftware/02_GUI_Widget_Guide.md/#networking) to learn how to stream data using UDP from the GUI. For this project, you will need to stream the EMG channel data from the Networking Widget. Your Networking settings should look as follows:
+
+![UDP Networking Widget Screenshot](../../assets/TutorialImages/UDP_ss.png)
+
+Once your UDP stream is running you will be able to use this EMG joystick for any use case of your choosing!
```

Comment:
Once your UDP stream is running, you will be able to use this EMG joystick for any use case! We are excited to see how our global Community uses this widget to create awesome projects and research!